components/Voorbeelden/aanvragen-file-upload.config.yml

```diff
@@ -13,10 +13,9 @@ context:
         id: panel1
         open: true
         form:
-          formModifier: form-horizontal
+          skipFormTag: false
```

Comment:
Het klinkt zo mal om in een `form` vervolgens met zoiets de `<form>` tag te skippen. Zou het niet logischer zijn om `components/Componenten/form/form.njk` juist in onderdelen te knippen:
- het form
- de eventuele fieldsets
- de groups
zodat je bij het modelleren van pagina's als deze zelf kunt bepalen op welk niveau je insteekt?